packages/usa-modal/src/index.js

```diff
@@ -170,104 +171,151 @@ function toggleModal(event) {
 }
 
 /**
- *  Builds modal window from base HTML
+ * Creates a placeholder with data attributes for cleanup function.
```

Comment:
Optional: 
It is not immediately clear to me why we need a placeholder, outside of the cleanup function needing it. Can we add detail to the comment about the purpose of this?

Reply:
Resolved in https://github.com/uswds/uswds/pull/5315/commits/c84d5ed04ad0b2cbf9d9dfec339df203d343da9c